### PR TITLE
Add configuration for htmltest

### PR DESCRIPTION
**Pull request by themr0c**
Add configuration files for https://github.com/wjdp/htmltest link checker.
Not for general usage yet, therefore no references in devfile or Dockerfile.
Just the minimum configuration to enable anyone to study the behavior of the tool in a local environment.
May later replace or complete linkchecker.

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

File: tools/podmanhtmltest.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
```
Comment (benoitf) [inline tools/podmanhtmltest.sh:1]: probably need to have copyright headers

File: tools/podmanhtmltest.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+set -ex
+
+podman run --rm -ti \
```
Comment (benoitf) [inline tools/podmanhtmltest.sh]: would be nice to have docker as a tool as well
(for example on macos I don't have a working podman)

File: tools/podmanhtmltest.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  -v "$(pwd):/test:Z" \
+  "${CHE_DOCS_IMAGE:-wjdp/htmltest}" \
```
Comment (benoitf) [inline tools/podmanhtmltest.sh]: where is referenced CHE_DOCS_IMAGE ?

Comment (themr0c) [reply tools/podmanhtmltest.sh]: It is not. It gives the ability to a user to change the image used.
But here it's a leftover of a copy-paste from another script.

File: tools/podmanhtmltest.sh (hunk 1 above)
Comment (themr0c) [reply tools/podmanhtmltest.sh:1]: Done.

File: tools/podmanhtmltest.sh (hunk 3 above)
Comment (themr0c) [reply tools/podmanhtmltest.sh]: Fixed

File: tools/podmanhtmltest.sh (hunk 2 above)
Comment (themr0c) [reply tools/podmanhtmltest.sh]: Done.

**Review by yhontyk — APPROVED** [review]

**Pull request merged**